binaries/pip/build_wheels.sh

```diff
@@ -57,7 +53,13 @@ mkdir output
 
 install_java_deps
 
-install_torch_deps
+cd ../../
+
+set +u
```

Comment:
This doesn't really seem as much of a fix as much as it's just masking over the original issue.

Reply:
the `install_torch_deps` function either installs the default torch packages (compatible to CUDA 10.2). For `CUDA10.1` the function expects `cuda101` to be supplied as an argument. 

Just removing the check to ignore the error in case the variable is not set.

Reply:
In fact torch packages are not required for building the `torchserve` binaries, hence refactored the code.